loadgenerator-manifests/loadgenerator.yaml

```diff
@@ -31,13 +31,14 @@ spec:
     spec:
       containers:
         - name: locust-main
-          image: gcr.io/stackdriver-sandbox-230822/sandbox/loadgenerator/gke:latest
+          # TODO(tugan): update this to official stack-driver container image, when updated
+          image: tugan0329/cloud-ops-sandbox-loadgen:latest
```

Comment:
When are you planning on updating this? Before merging?

Reply:
I don't have access to create a new image there. 

After this PR is approved, either someone can help me create one and I update it before merging, or we can have a different PR afterwards.

Reply:
I think you can leave it as `gcr.io/stackdriver-sandbox-230822/sandbox/loadgenerator/gke:latest`, and the image will be [updated automatically when you push to master](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/e1c33e6f6d7b8d43499942da52c762f94a3b20e7/.github/workflows/push-master.yml#L39), right?

Reply:
sounds good!

Reply:
@daniel-sanche 
If I change it to gcr.io/stackdriver-sandbox-230822/sandbox/loadgenerator/gke:latest, the E2E test for CI will run using the old latest image (because this PR hasn't been merged to master), but with the updated E2E test for the new API endpoint, thus fail. Should we send a follow up one line PR instead to change the path, once the new image is auto updated

Reply:
The intention of the CI is that it should be automatically building and testing the image based on what's in the repo HEAD, not the latest published image. (see [here](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/e1c33e6f6d7b8d43499942da52c762f94a3b20e7/.github/workflows/e2e-latest.yml#L54) where it builds a new tag, and [here](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/e1c33e6f6d7b8d43499942da52c762f94a3b20e7/.github/workflows/e2e-latest.yml#L63) where it re-writes the manifests to use it)

Although looking at the code, it's [updating `./kubernetes-manifests`](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/e1c33e6f6d7b8d43499942da52c762f94a3b20e7/.github/workflows/e2e-latest.yml#L63), but may be missing `loadgenerator-manifests`. You may have to update that part of the CI script if it's failing on your changes. But I don't think we should merge code to master pointing at a different container path